third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -1842,10 +1844,13 @@ fn parse_name_exp(context: &mut Context) -> Result<Exp_, Box<Diagnostic>> {
 // Parse the arguments to a call:
 //      CallArgs =
 //          "(" Comma<Exp> ")"
+// Result is a list <args>
 fn parse_call_args(context: &mut Context) -> Result<Spanned<Vec<Exp>>, Box<Diagnostic>> {
     let start_loc = context.tokens.start_loc();
-    let args = parse_comma_list(
+    consume_token(context.tokens, Tok::LParen)?;
```

Comment:
Is there any difference after this refactor?

Reply:
No, this was residue from the removed `..` notation.